### PR TITLE
Feature/0.1.3

### PR DESCRIPTION
**Pull request by kelly-vacasa**
This fixes issues in a certain tag only, so it should not be merged into master. Suggested workflow:

1. Create a new branch named `release/0.1.3` (or `release/0.1.3.1` if you'd prefer to increment) from the commit tagged as `0.1.3`.
```git checkout tags/0.1.3 -b release/0.1.3```
2. Merge the changes in this pull request into your new `release/0.1.3` branch. (Don't merge this pull request since it would go into the master branch.) 
  - It might be easiest to open a new pull request https://github.com/Vacasa/drf-jsonapi/compare/release/0.1.3...kelly-vacasa:feature/0.1.3 and manually merge it through the github GUI.
- or if you've already checked out `release/0.1.3`, this should do the same thing:
```git pull origin pull/41```
3. If you are renaming the `0.1.3` tag, delete it locally and remotely.
```git tag -d 0.1.3 && git push origin :refs/tags/0.1.3``` 
4. Tag the current commit as `0.1.3`.
```git tag 0.1.3```
- If you want to increment to the `0.1.3.1` tag, change the version number in `setup.py` to `0.1.3.1`, commit the changes, and tag the new commit as `0.1.3.1`.
5. Push the code changes as well as the new `0.1.3` tag to your new `release/0.1.3` branch
```git push --tags -u origin release/0.1.3```

Ideally the version numbers in `setup.py` for all tagged commits should be fixed. It might be easier to delete this pull request and run this script instead:
```#!/bin/bash

for current_tag in $(git tag --list); do
	git checkout -f tags/$current_tag -B release/$current_tag
	sed -i '' 's/0.1.0/'"$current_tag"'/g' setup.py
	git commit -a -m 'updated version from 0.1.0 to ${current_tag}'
	git tag -d $current_tag && git push origin :refs/tags/$current_tag && git tag $current_tag
	git push --tags -u origin release/$current_tag;
done

### CODE REVIEW

**Comment by tpansino** [comment]
Just noticed this PR, I think I have a way to keep it from happening in the future over in #43.

It might be easier just to merge in that change, cut a version, and start fixing this issue from there, rather than back-fixing all of the old versions which then have to be `git push --force`d up, and would change checksums so that tools like `pipenv` might complain during install. Just my 2 cents.

File: setup.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ def readme():
         'djangorestframework',
         'drf-nested-routers',
         'django-filter',
+        'django-filters',
```
Comment (tpansino) [inline setup.py]: @chrisbrantley tells me this isn't a required dependency, the `django-filter` package is the one actually being used (even though the code says `import django_filters`).

Chris, can you confirm? Also, you mentioned this was an optional feature of the library? Do we want to stick this under an `extras_require` setting in `setup.py`, and have them installed like `drf-jsonapi[with_filters]`?

File: setup.py [hunk 2]
```diff
@@ -7,7 +7,7 @@ def readme():
 
 setup(
     name='drf-jsonapi',
-    version='0.1.0',
+    version='0.1.3',
```
Comment (tpansino) [inline setup.py]: This is set via `setuptools_scm` now, hence the conflict with `setup.py` that GitHub is mentioning. Need to decide if we want to continue with that or use a different workflow. @kelly-vacasa and I will discuss today.

File: setup.py (hunk 1 above)
Comment (chrisbrantley) [reply setup.py]: Correct. `django-filter` and `drf-yasg` are optional dependencies and not required for the framework to function. I love the idea of using `extras_require`.

`drf-nested-routers` is no longer needed at all.

**Comment by kelly-vacasa** [comment]
Since my changes involve rewriting the tagged release history, maybe it's best to close the pull request without merging and just discuss how things will be handled in future releases.

**Comment by tpansino** [comment]
I've captured the core changes from this PR and several offline discussions about it over in #45. @morgandenning , could you close this out? And then we'll need to work on getting that puppy merged again.

**Pull request closed without merging**